charts/infra-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
-version: 0.130.0
+version: 0.131.0
```

Comment:
Could yo also add in this file 
```
annotations:
  artifacthub.io/changes: |
```
The list of changes this change is triggering? feel free to remove the old ones, so we only have the most recent changes per version.